docs/usage/maintenance.rst

```diff
@@ -0,0 +1,28 @@
+Maintenance
+***********
+
+collective.solr comes with a set of command line options to activate or deactivate Solr and update the Solr index.
+
+Activate Solr::
+
+    bin/instance solr_deactivate
```

Comment:
`     bin/instance solr_activate` you meant? :)

Reply:
@gforcada right. Good catch. Thanks. :)